frontend/express/public/javascripts/countly/vue/components/input.js

```diff
@@ -748,7 +748,7 @@
             mode: {type: String, default: 'single-list'}, // multi-check,
             autoCommit: {type: Boolean, default: true},
             disabled: { type: Boolean, default: false},
-            width: { type: [Number, Object], default: 400},
+            width: { type: [Number, Object, String], default: 400},
```

Comment:
That 400 is most likely there because of some older design decision - and it was just decided as for default value.
I don't see anything bad with adding 'String'